libs/atomvmlib/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Copyright 2023 Fred Dushin <fred@dushin.net>
+#
+# SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
+rebar.config
+rebar.lock
```

Comment:
shouldn't we version lock files?

Reply:
Probabaly not in this case, because in most cases the dependencies are internal to this project, and we do not version them; nod do we put them in GitHub repositories structured in such a way that they can be pulled down.